wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -581,6 +581,38 @@ public WrapperCommand withName(String name) {
     return wrapper;
   }
 
+  /**
+   * Throws an error if a parallel group already shares
+   * one or more requirements with a command
+   * that will be added to it.
```

Comment:
As an FYI, this will need to be formatted (and since your `main` branch is a PR branch, /format on your PRs won't work). You can look at [the comment-command .yml file](https://github.com/wpilibsuite/allwpilib/blob/44a45d44e2686d2161374d39154417d62300e7db/.github/workflows/comment-command.yml#L45) to see what steps you should run to format. (`pip3 install wpiformat==2024.45`, `wpiformat`, and `./gradlew spotlessApply`)

Reply:
yeah forgot to run /format